### PR TITLE
Drop support for Pimcore 10

### LINKED ISSUE #412: Drop Pimcore 10 Support
### Feature description

Basically to support https://github.com/pimcore/customer-data-framework/pull/410

### PR DESCRIPTION
**Pull request by aryaantony92**
Resolves #412

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,37 +15,37 @@
...
+    "symfony/dependency-injection": "^6.1",
+    "symfony/event-dispatcher": "^6.1",
+    "symfony/http-foundation": "^6.1",
+    "symfony/http-kernel": "^6.1",
+    "symfony/options-resolver": "^6.1",
+    "symfony/routing": "^6.1",
+    "symfony/security-bundle": "^6.1",
+    "symfony/security-core": "^6.1",
+    "symfony/security-http": "^6.1",
+    "symfony/stopwatch": "^6.1"
```
Comment (blankse) [inline composer.json]:
```suggestion
    "symfony/asset": "^6.2",
    "symfony/config": "^6.2",
    "symfony/console": "^6.2",
    "symfony/dependency-injection": "^6.2",
    "symfony/event-dispatcher": "^6.2",
    "symfony/http-foundation": "^6.2",
    "symfony/http-kernel": "^6.2",
    "symfony/options-resolver": "^6.2",
    "symfony/routing": "^6.2",
    "symfony/security-bundle": "^6.2",
    "symfony/security-core": "^6.2",
    "symfony/security-http": "^6.2",
    "symfony/stopwatch": "^6.2"
```

File: .github/workflows/codeception.yml [hunk 2]
```diff
@@ -29,7 +29,6 @@ jobs:
         strategy:
             matrix:
                 include:
-                    - { php-version: 8.0, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: lowest, experimental: false }
```
Comment (dvesh3) [inline .github/workflows/codeception.yml:32]: we should still run lowest dependencies tests

Comment (aryaantony92) [reply .github/workflows/codeception.yml:32]: With php-version 8.1, right ?

Comment (dvesh3) [reply .github/workflows/codeception.yml:32]: yes, since this is the min requirement in composer.json now.

File: composer.json [hunk 3]
```diff
@@ -15,37 +15,37 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "php": "^8.0 || ^8.1",
+    "php": "^8.1",
```
Comment (blankse) [inline composer.json]: See comments in https://github.com/pimcore/pimcore/pull/14080
```suggestion
    "php": "~8.1.0 || ~8.2.0",
```

Comment (aryaantony92) [reply composer.json]: @blankse  Thanks! Done in [f3c4204](https://github.com/pimcore/customer-data-framework/pull/416/commits/f3c42045784f6d9b1d9c20fa1590386408d310ce)

**Pull request merged**